### ISSUE TITLE
Volume slider not functionning as a slider

### ISSUE THREAD

**Issue by LostToPixels** [Bionus/imgbrd-grabber#3242]
**Bug description**

Volume slider does not affect how loud the audio is unless set to minimum where no audio plays at all.

**Steps to reproduce**

1. Open any post with a "sound" tag
2. Click and drag the slider around

**Expected behavior**

Volume slider should fade from full volume all the way down to silence in a linear manner.

**Context**

[main_2024.07.05-20.17.59.210.log](https://github.com/user-attachments/files/16114485/main_2024.07.05-20.17.59.210.log)
Issue happens after a fresh install, without any changes to settings.

**Screenshots**

![Screenshot 2024-07-05 202842](https://github.com/Bionus/imgbrd-grabber/assets/121353365/34d34c79-3151-4ff3-8b2c-7af12660d046)

**System information**

- Windows 11 v23H2
- Grabber version 7.12.2

**Additional context**

Really have no clue what could be causing this. Doubt it would be related to having uninstalled any windows default media viewers and installing K-Lite Codec pack instead but that's all I can think that could be something to go off of.

**Comment by VeggieJay** [comment]
Have the same issue on W10
Volume control does seem to work between like 0-5% on the slide but 6%-100% doesn't change volume at all

**Comment by Bionus** [comment]
Indeed, seems like the slider goes from 0 to 100 while it should go from 0 to 1 instead 👍

**Comment by LostToPixels** [comment]
for those still having this issue, use the [nightly release](https://github.com/Bionus/imgbrd-grabber/releases/nightly)
